azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/ApplicationGatewaySslCertificate.java

```diff
@@ -28,6 +28,14 @@
      */
     String publicData();
 
+    /**
+     * Get Secret Id of (base-64 encoded unencrypted pfx) 'Secret' or 'Certificate'
+     object stored in KeyVault.
```

Comment:
Missed `*` for the comment

Reply:
It's a little confused, since I searched for the whole project and found that comments in one sentence didn't use a `*` in newline. Such as: https://github.com/Azure/azure-libraries-for-java/blob/3f786b03a1d7b474bd7aed8a199980f994191f36/azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/AutoHealActions.java#L79, https://github.com/Azure/azure-libraries-for-java/blob/3f786b03a1d7b474bd7aed8a199980f994191f36/azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/AzureBlobStorageApplicationLogsConfig.java#L81